### PR TITLE
Bug 1506144 - Access bugzilla security bug metadata via STMO

### PR DESCRIPTION
**Pull request by dylanwh**
This is the current expanded schema. I'd like to know what additional fields we want before I start the process of getting it signed off. @psiinon  @mars-f 
If the below is too verbose, consider looking at [the source](https://github.com/mozilla-bteam/bmo/blob/368fa33a172352c9c7681448934509a8ed5bbc67/Bugzilla/Report/Ping/Simple.pm#L23-L31) which is defined using [joi](https://github.com/hapijs/joi)

```json
{
   "additionalProperties" : false,
   "properties" : {
      "bug_id" : {
         "minimum" : 1,
         "type" : "integer"
      },
      "bug_status" : {
         "type" : "string"
      },
      "groups" : {
         "items" : {
            "type" : "string"
         },
         "type" : "array"
      },
      "keywords" : {
         "items" : {
            "type" : "string"
         },
         "type" : "array"
      },
      "priority" : {
         "type" : "string"
      },
      "resolution" : {
         "type" : "string"
      },
      "severity" : {
         "type" : "string"
      }
   },
   "required" : [
      "bug_status",
      "keywords",
      "priority",
      "groups",
      "bug_id",
      "severity"
   ],
   "type" : "object"
}
```

### CODE REVIEW

**Comment by dylanwh** [comment]
Revised list: 

```
alias
assigned_to
blocks
classification
comment_count
component
creation_time
creator
depends_on
dupe_of
duplicates
flags
groups
id
is_cc_accessible
is_confirmed
is_creator_accessible
is_open
keywords
last_change_time
mentors
op_sys
platform
priority
product
qa_contact
resolution
see_also (array of urls)
severity
bug_status
target_milestone
url
version
votes
whiteboard
```

File: Bugzilla/Report/Ping/Simple.pm [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+  my $options = {
+    order_by => 'me.bug_id',
+  };
+  return $bugs->search($query, $options);
+}
+
+sub prepare {
+  my ($self, $bug) = @_;
+  my $doc = {
+    bug_id       => 0 + $bug->id,
```
Comment (dylanwh) [inline Bugzilla/Report/Ping/Simple.pm]:
```suggestion
    bug_id       => $bug->id,
```

File: Bugzilla/App/Command/report_ping.pm [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+  }
+}
+
+1;
+
+__END__
+
+=head1 NAME
+
+Bugzilla::App::Command::report_ping - descriptionsend a report ping to a url';
```
Comment (dklawren) [inline Bugzilla/App/Command/report_ping.pm]: s/descriptionsend/send/

File: Bugzilla/App/Command/report_ping.pm [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+
+=head1 ATTRIBUTES
+
+L<Bugzilla::App::Command::report_ping> inherits all attributes from
+L<Mojolicious::Command> and implements the following new ones.
+
+=head2 description
+
+  my $description = $report_ping->description;
+  $rereport r      = $re$port_ping->description('Foo');
```
Comment (dklawren) [inline Bugzilla/App/Command/report_ping.pm]: Fix this line

File: Bugzilla/App/Command/report_ping.pm [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+
+Short description of this command, used for the command list.
+
+=head2 usage
+
+  my $usage = $report_ping->usage;
+  $report_ping  = $report_ping->usage('Foo');
```
Comment (dklawren) [inline Bugzilla/App/Command/report_ping.pm]: extraneous white space

**Pull request merged**